tests/test_bootstrap_cli.py

```diff
@@ -109,20 +109,17 @@ def test_make_config(tmp_path):
 
     assert (tmp_path / "copilot").exists()
 
-    def real_line_breaks(input: any) -> str:
-        return str(input).replace("\\n", "\n")
-
     with open(str(tmp_path / "copilot/.workspace")) as workspace:
         assert workspace.read() == "application: test-app"
 
-    with open(str(tmp_path / "copilot/environments/test/manifest.yml"), "rb") as test:
-        assert real_line_breaks(test.read()) == real_line_breaks(test_environment_manifest)
+    with open(str(tmp_path / "copilot/environments/test/manifest.yml"), "r") as test:
```

Comment:
Was it the opening in binary mode that made it do the line breaks as "\n" strings?

Apparently "r" is the default, so you can drop those.